packages/ketcher-core/src/application/ketcher.ts

```diff
@@ -111,41 +115,36 @@ export class Ketcher {
   // TEMP.: getting only dearomatize-on-load setting
   get settings() {
     const options = this.#editor.options()
+    const result = Object.entries(allowedApiSettings).reduce(
+      (acc, [apiSetting, clientSetting]) => {
+        if (clientSetting in options) {
+          return { ...acc, [apiSetting]: clientSetting }
+        }
+        return acc
+      },
+      {}
+    )
 
-    if ('dearomatize-on-load' in options) {
-      return {
-        'general.dearomatize-on-load': options['dearomatize-on-load']
-      }
-    }
-    if ('ignoreChiralFlag' in options) {
-      return {
-        'general.ignore-no-chiral-flag': options.ignoreChiralFlag
```

Comment:
@ansivgit I dont think this needs to be in server-style, I changed "general.ignore-no-chiral-flag" to "ignoreChiralFlag" here - do you think this is OK?

Reply:
Ok